src/platform/silabs/SilabsConfig.cpp

```diff
@@ -43,13 +46,19 @@ static StaticSemaphore_t nvm3_SemStruct;
 void nvm3_lockBegin(void)
 {
     VerifyOrDie(nvm3_Sem != NULL);
+#ifdef BRD4325A
+    __disable_irq();
```

Comment:
need to be discussed internally. I don't believe we should disable IRQ in our multi-thread context. We need clarification on what causes the device to reboot.